FreeAPS/Sources/Localizations/Shortcuts/en.lproj/AppShortcuts.strings

```diff
@@ -0,0 +1,36 @@
+/*
+  AppShortcuts.strings
```

Comment:
Given the comment `has to be redone after finalizing the shortcuts` in this file – is this final and okay to merge?

Reply:
@avouspierre Can you answer this please?

Reply:
![CleanShot 2024-06-14 at 20 03 43@2x](https://github.com/nightscout/Trio/assets/2100083/ad23415d-8460-402f-9b67-45f5bc9b7904)
These are actually not used

Reply:
@dnzxy Can you look into these localised strings for me?

Reply:
update shortcuts strings

Reply:
Tnx mate!